cantusdb/csv/fetch.py

```diff
@@ -0,0 +1,32 @@
+"""
+fetch the CSV exports
+"""
+
+import json
+import os
+import time
+import requests
+
+# URL of the file to download
+SOURCE_PATH = os.path.join(
+    os.path.dirname(__file__), "../data/mappings/sources_short.json"
```

Comment:
you can just use the relative path...

Reply:
I always wanted to do this, but I searched and I don't see any better ways of using relative path..